user-management.yaml

```diff
@@ -438,35 +308,28 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/GenericError'
-        "409":
+        "404":
           description: |
-            Conflict  
-            types: key duplicate
+            Not Found  
+            types: key not found
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/GenericError'
-        "422":
-          description: |
-            Unprocessable Entity  
-            types: validation error
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/DetailedError'
+    
+  /role/{username}:  
```

Comment:
can/should we now go back to 
`/users/{username}/role`
without ambiguous routes?

Reply:
We can, and we did. Thank you for the suggestion.